farmerbot/cmd/run.go

```diff
@@ -15,7 +19,21 @@ var runCmd = &cobra.Command{
 		if len(cmd.Flags().Args()) != 0 {
 			return fmt.Errorf("'run' and %v cannot be used together, please use one command at a time", cmd.Flags().Args())
 		}
+		autoApprove, err := cmd.Flags().GetBool("auto-approve")
+		if err != nil {
+			return err
+		}
+		if !autoApprove {
+			answer, err := disclaimerPrompt()
+			if err != nil {
+				return err
+			}
+			if answer == "no" {
+				log.Info().Msg("Farmer bot is exiting ....")
```

Comment:
make it `Farmerbot` to match other words

Reply:
done